src/types/ChargingStation.ts

```diff
@@ -278,4 +278,5 @@ export enum ChargerVendor {
   LEGRAND = 'Legrand',
   ATESS = 'ATESS',
   MENNEKES = 'MENNEKES',
+  SAPLABSFRANCE = 'SAP Labs France Caen',
```

Comment:
SAP Labs France here too ;-)

Reply:
I can't, it's used as a mapping between the charging station and the integration.
If I change the value in the firmware, boot notification will be rejected for existing charging stations.

![image](https://user-images.githubusercontent.com/9685252/101691379-e522aa80-3a6e-11eb-9a97-d8e74c0d99d2.png)

Reply:
Despite I'm not happy with your choice, now that it is productive, you're right, it cannot be changed.
Just rename the constant in snake case SAP_LABS_FRANCE.
Thanks!

Reply:
Don't forget to move the issue back to the right pipeline in Zenhub

Reply:
Renamed. The issue is in the review column, is it ok ?

Reply:
> The issue is in the review column, is it ok ?

No (check ZenHub)
![image](https://user-images.githubusercontent.com/25266945/101853583-1f1fa980-3b60-11eb-944e-72394073a0f2.png)

![image](https://user-images.githubusercontent.com/25266945/101853492-f3042880-3b5f-11eb-9b43-5a8731b0e1f0.png)

Reply:
I was not on the correct board

On my side:
![image](https://user-images.githubusercontent.com/9685252/101924024-6f365480-3bd0-11eb-8d93-d394494b42a3.png)

Reply:
Indeed, I removed them.
Thanks for reporting.